index.bs

```diff
@@ -1294,7 +1295,8 @@ does not change between operations but instead remains the same for the lifetime
 by the authenticator during the [=authenticatorGetAssertion=] operation, by verifying that the RP ID associated with the
 requested credential exactly matches the RP ID supplied by the client.
 
-The `TUP` flag SHALL be set if and only if the authenticator detected a user through an authenticator specific gesture. The
+The `TUP` flag SHALL be set if and only if the authenticator detected a user through an authenticator specific gesture. The 
```

Comment:
s/TUP/UP/ ?  (ie, "User Present")

Also, given the language here, we seem to be saying that if UV has been accomplished, **BOTH** the UP (nee 'TUP') and UV flags would be set, yes?

I.e., UP is set if the authenticator is able to confirm user presence in **any** fashion, and UV is _also_ set if that fashion is one able to verify that it is the "same user" as in prior interactions (e.g., credential creation).

Yes? 

If so, this language ought to be polished appropriately (in both the above and below lines). 

BTW, this is the way the FIDO User Verification Methods flags work -- USER_VERIFY_PRESENCE is always set, and then other flags MAY be set in order to signal finer-grained attributes of the interaction, see: https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-registry-v1.1-id-20170202.html#user-verification-methods